fiftyone/utils/torch.py

```
@@ -649,11 +650,72 @@ def __call__(self, d):
     @property
     def required_keys(self):
         return ["filepath"]
 
 
+class VideoFrameGetItem:
+    """Wrapper around model's get_item that injects pre-loaded frames.
+
+    This replaces filepath loading but preserves all other get_item logic.
+    """
+
+    def __init__(self, base_get_item=None):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)